src/anbox/dbus/ApplicationManager.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
IMO, a plain description of an interface shouldn't be copyrightable, but my opinions likely doesn't matter on legal terms. You may want to add some GPL header or a SPDX-License-Identifier.

Reply:
Any idea how SPDX-License-Identifier would look like in XML document? I may add just a plain text comment though (should be something more permissive than GPL I think).

Reply:
that's a good point.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!-- SPDX-License-Identifier: MIT -->
```

should do, but @morphis should also say if he's ok with MIT license for that. Note that it's also possible to set it to something like `(GPL-2.0 OR MIT)`.